dali/python/nvidia/dali/plugin/paddle.py

```diff
@@ -0,0 +1,445 @@
+# Copyright (c) 2017-2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
I think you can add PaddlePaddle to shi copyright here in all oiher examples (RN50, SSD, TSM).

Reply:
done.